src/crimson/net/SocketMessenger.cc

```diff
@@ -134,6 +134,9 @@ seastar::future<> SocketMessenger::start(Dispatcher *disp) {
 crimson::net::ConnectionRef
 SocketMessenger::connect(const entity_addr_t& peer_addr, const entity_name_t& peer_name)
 {
+  if (stopping)
+    throw std::system_error(
+	make_error_code(crimson::net::error::network_aborted));
```

Comment:
nit, might need add related comment to `crimson::net::Messenger::connect()` interface.